modules/@apostrophecms/ui/ui/apos/mixins/AposCellMixin.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  methods: {
+    // Fetch a field from the published version of the item, if available and
+    // defined, otherwise from the draft version of the item
+    displayValue(fieldName) {
+      const publishedDoc = this.item._publishedDoc;
+      return (publishedDoc && (publishedDoc[fieldName] !== undefined) && publishedDoc[fieldName]) || this.item[fieldName];
```

Comment:
Why do you need the `(publishedDoc[fieldName] !== undefined)` step?

Reply:
I'm falling back to draft properties when the published doc truly doesn't have them at all, which increases the set of use cases that can be covered without one-off logic. (There is a separate bug currently breaking submitted.by and submitted.at, but I'll be addressing that elsewhere after this merges)